src/style/content/conversation/message-quote.less

```
@@ -72,9 +72,10 @@
       pre {
         overflow: auto;
         max-width: var(--conversation-message-asset-width);
-        padding: 16px;
+        padding: 12px;
         border-radius: 12px;
         background: var(--foreground-fade-8);
+        font-size: 12px;
```

```suggestion
        font-size: var(--font-size-small)
```

missed that before apparently

---

done :)